portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,13 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+// Cycles through 3 different pictures of myself.
+let pictureNumber = 0;
+function cyclePictures() {
+  let headerSelfie = document.getElementById("header-selfie");
+  const imageNames = ["campus", "suit", "mainSelfie"];
+  headerSelfie.src = "images/" + imageNames[pictureNumber] + ".jpg";
+  pictureNumber++;
+  if (pictureNumber === 3) pictureNumber = 0;
```

Comment:
> Braces are required for all control structures (i.e. if, else, for, do, while, as well as any others), even if the body contains only a single statement. The first statement of a non-empty block must begin on its own line.

See https://google.github.io/styleguide/jsguide.html#formatting-braces-all

```javascript
if (pictureNumber === 3) {
  pictureNumber = 0;
}
```

Optionally, you can do this in a single line with a modulo operator.

```javascript
pictureNumber %= 3;
```

Or remove this line and include it above:

```javascript
headerSelfie.src = `images/${imageNames[pictureNumber++ % imageNames.length]}.jpg`;
```

Reply:
Oh okay, I chose the third option.